platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -140,6 +142,13 @@ public void init() throws Exception {
         return export(Paths.get(exportDirectory));
     }
 
+    @Override
+    public Collection<Describable> getOptionalMigratableInfo() {
+        return dataMigratables.stream()
```

Comment:
Tried it out. Not possible. Can't cross-cast a list of different types. Need to cast each entity separately.